process/archiving.md

```diff
@@ -2,29 +2,40 @@
+CNCF TOC may also look at a variety of signals, including but not limited to:
+* [CNCF Project health dashboard](https://projecthealth.cncf.io/)
+* If the project would currently meet criteria for acceptance to CNCF
+* Requests for archival by anyone in the wider CNCF community
+* Security posture of the project, in particular speedy reaction to and resolution of vulnerability reports
+* Project meetings schedules and notes
+* Public mailing list activity
+* Adoption stats
 
-## Voting Process
+Automated systems and the references above can trigger a review by the TCO, for example a project might be very mature and close to feature complete, which would be different from general inactiveness. The archival process itself remains manual with close attention to the details by TOC.
 
-To archive a project:
+## Archiving Process
 
-* A proposal must be put forth to the TOC repo
-* The TOC will inform the project maintainers, CNCF End-User community and wider community of all archiving proposals
-* The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
-* A vote must be finalized with 2/3 approval from the TOC
+This is a multi-step process with ample time for projects to react, discuss, and improve built in. Being slow and deliberate is a bug, not a feature. The process can be stopped by TOC consensus at any time until a formal vote has been announced.
 
-## Archiving Process
+1. Archiving review is triggered (see above).
+2. TOC opens an issue inquiring about the health of the project. While there's not fixed timeline, it will be in the order of months.
+3. A formal archival request is created in https://github.com/cncf/toc/issues and the issue tracker of the member project, cross-linking both.
+4. TOC informs project maintainers and the end user community. This step will usually be delegated to CNCF staff.
+5. A mandatory two week discussion phase begins
+6. A TOC vote is announced; the voting duration is two weeks. A 2/3 majority in favor of archiving is required.
+
+## Archived Status
 
 What does archiving for a CNCF project mean?
 
 * CNCF will no longer provide support for the project via service desk
-* CNCF will list archived projects online
+* CNCF will list the archived project at https://www.cncf.io/archived-projects/
 * Trademarks and domain names of archived projects are still hosted by the CNCF and the Linux Foundation
 * CNCF can provide services such as documentation updates to help transition users.
 * Other CNCF marketing activities will no longer be provided for the project
 
 ## Reactivating an Archived Project
```

Comment:
Can it be made explicit here that -  project source code repositories will continue to be maintained by the respective project community and CNCF will not force them to make the project read-only.

Reply:
Good point. Like so?

https://github.com/cncf/toc/pull/1050/commits/75fac3d892412fb4c7a2829d40b09d6ac5d6c2ab